managed/CounterStrikeSharp.API/Modules/Utils/CanAcquire.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace CounterStrikeSharp.API.Modules.Utils;
+
+public enum AcquireMethod : int
+{
+    PickUp = 0,
+    Buy,
+};
+
+public enum AcquireResult : int
+{
+    Allowed = 0,
+    InvalidItem,
+    AlreadyOwned,
+    AlreadyPurchased,
+    ReachedGrenadeTypeLimit,
+    ReachedGrenadeTotalLimit,
+    NotAllowedByTeam,
+    NotAllowedByMap,
+    NotAllowedByMode,
+    NotAllowedForPurchase,
+    NotAllowedByProhibition,
+};
```

Comment:
instead of `CanAcquire.cs`, create `AcquireResult.cs` and `AcquireMethod.cs` with the correspondive values in them. also dont forget the license text at the top